webapp/channels/src/components/threading/virtualized_thread_viewer/virtualized_thread_viewer.tsx

```diff
@@ -350,6 +353,12 @@ class ThreadViewerVirtualized extends PureComponent<Props, State> {
         const isLastPost = itemId === this.props.lastPost.id;
         const isRootPost = itemId === this.props.selected.id;
 
+        const post = useSelector((state: GlobalState) => getPost(state, this.props.selected.id));
+        const getThreadOrSynthetic = useMemo(makeGetThreadOrSynthetic, [post.id]);
+        const thread = useSelector((state: GlobalState) => getThreadOrSynthetic(state, post));
+
+        const {reply_count: totalReplies = 0} = thread;
```

Comment:
If we use only the reply count from the `thread`, let's get it from inside the selector. That way we avoid unneeded re-renders.